Src/MoneyFox.Uwp/MoneyFox.Uwp.csproj

```diff
@@ -37,6 +37,8 @@
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
     <CodeAnalysisRuleSet>MoneyFox.Uwp.ruleset</CodeAnalysisRuleSet>
+    <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
```

Comment:
can you please deactivate these two for the pull request?

Reply:
They should be removed now